controllers/grafana/grafana_controller.go

```diff
@@ -188,17 +190,19 @@ func (r *ReconcileGrafana) Reconcile(ctx context.Context, request reconcile.Requ
 	}
 
 	// Get the actions required to reach the desired state
-
+	log.V(1).Info("Create GrafanaReconciler and determine desiredState...")
 	reconciler := NewGrafanaReconciler()
 	desiredState := reconciler.Reconcile(currentState, cr)
 
+	log.V(1).Info("Determined desiredStates - starting actionRunner")
 	// Run the actions to reach the desired state
 	actionRunner := common.NewClusterActionRunner(ctx, r.Client, r.Scheme, cr)
 	err = actionRunner.RunAll(desiredState)
 	if err != nil {
 		return r.manageError(cr, err, request)
 	}
 
+	log.V(1).Info("Now processing configMaps...")
```

Comment:
The reconciler loop is already rather chatty. Is it worth writing this log for every run?